docs/installation.md

```diff
@@ -91,12 +90,24 @@ You may also start Kuzzle by using [PM2](https://github.com/Unitech/pm2):
 $ pm2 start app-start.js
 ```
 
-### Change RabbitMQ and Elasticsearch hosts
 
-If you are not running RabbitMQ and Elasticsearch on localhost, you can configure host and port:
+### Change external services hosts
+
+If you are some external service in another host than localhost, you can configure host and port
```

Comment:
If you are running some of the service(s) externally, you can configure their host and port using some environment variables: